packages/components/src/components/tag/tag-styles.ts

```diff
@@ -100,21 +52,20 @@ export const getComponentCss = (tagColor: TagColor, isFocusable: boolean, theme:
         display: 'flex',
         alignItems: 'center',
         position: 'relative',
-        height: pxToRemWithUnit(24),
-        padding: `0 ${pxToRemWithUnit(6)}`,
-        borderRadius: pxToRemWithUnit(4),
+        padding: '5px 9px',
+        borderRadius: borderRadiusSmall,
         background: backgroundColor,
         color: primaryColor,
         ...textXSmallStyle,
         whiteSpace: 'nowrap',
-        ...(isFocusable && {
-          transition: getTransition('color'),
-          ...hoverMediaQuery({
+        transition: `${getTransition('background-color')}`,
```

Comment:
backticks are unnecessary here

Reply:
In addition, shouldn't it be part of the `isFocusable` condition?

Reply:
true, fixed it